modules/osd-aws-privatelink-architecture.adoc

```diff
@@ -1,21 +1,23 @@
 [id="osd-aws-privatelink-architecture.adoc_{context}"]
 = PrivateLink architecture
 
-The Red Hat managed infrastructure responsible for creating PrivateLink clusters is hosted on completely private subnets. The only connection made between Red Hat and the customer provided infrastructure is through PrivateLink VPC Endpoints. 
+The Red Hat managed infrastructure that creates AWS PrivateLink clusters is hosted on completely private subnets. The only connection made between Red Hat and the customer provided infrastructure is through PrivateLink VPC Endpoints.
 
-The following diagram shows an example of a multiple availability zone (Multi-AZ) PrivateLink cluster deployed only on private subnets. 
+The following diagram shows an example of a multiple availability zone (Multi-AZ) PrivateLink cluster deployed only on private subnets.
 
 .Multi-AZ PrivateLink cluster deployed on private subnets
 
 image::156_OpenShift_ROSA_Arch_0621_privatelink.svg[Multi-AZ PrivateLink cluster deployed on private subnets]
 
-AWS provides multiple reference architectures. Here are three examples:
+== AWS architecture for internet access without an internet gateway
+
+AWS provides multiple reference architectures that can be useful to customers when planning how to set up a configuration that uses PrivateLink. Here are three examples:
 
 * VPC with a private subnet and AWS Site-to-Site VPN access.
 +
 This configuration enables you to extend your network into the cloud without exposing your network to the internet.
 +
-It contains a virtual private cloud (VPC) with a single private subnet and a virtual private gateway to enable communication with your network over an Internet Protocol Security (IPsec) VPN tunnel. There is no internet gateway to enable communication over the internet. 
+It contains a virtual private cloud (VPC) with a single private subnet and a virtual private gateway to enable communication with your network over an Internet Protocol Security (IPsec) VPN tunnel. There is no internet gateway to enable communication over the internet.
```

Comment:
Consider this edit:  To enable communication with your network over an Internet Protocol Security (IPsec) VPN tunnel, this configuration contains a virtual private cloud (VPC) with a single private subnet and a virtual private gateway. Communication over the internet does not use an internet gateway. [Also, consider using a standard approach to address the customer. Either use _your_ or _customer_ but not both.